src/KintoID.sol

```diff
@@ -219,6 +227,9 @@ contract KintoID is
         Metadata storage meta = _kycmetas[_signatureData.signer];
         meta.mintedAt = 0;
         meta.updatedAt = 0;
+
+        // Set sanctionedAt to the current block timestamp
```

Comment:
is this needed? When we burn we dont care about this anymore

Reply:
Yes. Otherwise how we'll unblock a user in 3 days, if the burn is not confirmed?